### ISSUE TITLE
EIP Support

### ISSUE THREAD

**Issue by ambsw-technology** [cfn-modules/nlb#5]
One of the reasons to use an NLB (over, or in addition to, an ALB) is the ability to attach an EIP using [subnet mappings](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-elasticloadbalancingv2-loadbalancer-subnetmapping.html).  I assume the EIP can't be used for anything else so there's no harm creating it within the template and exporting it.

**Comment by michaelwittig** [comment]
The benefit of creating the EIP outside the nlb module is that you can remove the NLB without losing the EIPs. If you use the EIP for IP whitelisting in external systems you don't want the EIPs to change ever. So I guess we should add an eip module as well :)

**Comment by ambsw-technology** [comment]
You're right that some people will need the EIP externalized to preserve the addresses.  Even if the module had some sort of "default" EIP generation, it will make sense to use/nest the same EIP template.
